### PR TITLE
Link to repo updated

### LINKED ISSUE #244: chore(docs, landing-page): update links to repo
We moved! Now the links need updating 😅

✨ Find and update links to the repository in our docs and landing-page apps

### PR DESCRIPTION
**Pull request by maddy020**
## type(scope): description
chore(docs, landing-page): update links to repo 

Fixes #244  

Description of changes: 
- Github root link changed

Tag a reviewer: @

Tasks:
- [x] I have ran the build command to make sure apps are working: `npm run build`
- [x] I have ran the tests to make sure nothing is broken: `npm run test`
- [x] I have ran the linter to make sure code is clean: `npm run lint`
- [x] I have reviewed my own code to remove changes that are not needed

### CODE REVIEW

File: apps/docs/src/pages/en/introduction.md [hunk 1]
```diff
@@ -34,24 +34,25 @@ However we also see that Astro has given us a good baseline framework to potenti
...
-| data-grid | 🛠 | a dynamic data grid of values |
-| themes | 🛠 | easy-to-use, accessible, consistent cross-browser styles |
-| viz | 🛠 | data visualization that emits and responds to events |
+| Package                                                                                              | Release notes                                                                                     | Description                                              |
+| ---------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
+| [form](/en/api/form/form-component)                                                                  | [![npm](https://img.shields.io/npm/v/@astro-reactive/form)](./packages/form/RELEASE.md)           | a dynamic form which can be modified programmatically    |
```
Comment (Preet-Sojitra) [inline apps/docs/src/pages/en/introduction.md:53]: image
I opened this in rich diff mode by clicking on this icon then I clicked the  `form`  package link, it is giving 404 error. Please check it once.
I guess it should be replaced with this `https://github.com/astro-reactive/astro-reactive/blob/main/packages/form/README.md` @ayoayco ?

All other links are working fine!!

**Review by Preet-Sojitra — CHANGES_REQUESTED** [review]

File: apps/docs/src/pages/en/introduction.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ We will also make this clear in our library documentation as we see them.
 
 | Package                                                                                              | Release notes                                                                                     | Description                                              |
 | ---------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- | -------------------------------------------------------- |
-| [form](/en/api/form/form-component)                                                                  | [![npm](https://img.shields.io/npm/v/@astro-reactive/form)](./packages/form/RELEASE.md)           | a dynamic form which can be modified programmatically    |
+| [form](/en/api/form/form-component)                                                                  | [![npm](https://img.shields.io/npm/v/@astro-reactive/form/README.md)](./packages/form/RELEASE.md)           | a dynamic form which can be modified programmatically    |
```
Comment (ayoayco) [inline apps/docs/src/pages/en/introduction.md:53]: this change to the `shields.io` URL from your latest commit is not right, it should not have `README.md`

**Review by ayoayco — CHANGES_REQUESTED** [review]

File: apps/docs/src/pages/en/introduction.md (hunk 1 above)
Comment (ayoayco) [reply apps/docs/src/pages/en/introduction.md]: It should be fine once this is merged. No need to change :)

Comment (ayoayco) [reply apps/docs/src/pages/en/introduction.md]: Woops I just noticed this is not the repo README but on our docs site. @Preet-Sojitra you are right.

**Review by ayoayco — APPROVED** [review]

**Pull request merged**